gems/pending/util/miq_winrm.rb

```diff
@@ -35,6 +35,18 @@ def run_powershell_script(script)
   rescue WinRM::WinRMAuthorizationError
     $log.info "Error Logging In to #{hostname} using user \"#{username}\"" unless $log.nil?
     raise
+  rescue WinRM::WinRMWMIError => error
+    $log.debug "Error Running Powershell on #{hostname} using user \"#{username}\": #{error}" unless $log.nil?
```

Comment:
Not a problem.  Its "unless $log.nil" elsewhere in the file as well.  I'll change all even though its not directly related.  Unless someone has an issue....  :-)